client.go

```diff
@@ -191,7 +191,7 @@ func (c *Client) getHTTP(ctx context.Context, uri string) (Result, error) {
 		ntpTime = time.Time(result.Time) // fallback htptime
 	}
 	delay := end.Sub(start)
-	offset := ntpTime.Sub(start) - delay/2
+	offset := ntpTime.Sub(end) + delay/2
```

Comment:
The wall clock may be changed after sending the request.
So, we use recent wall clock time.